src/state/actions/window.js

```diff
@@ -69,32 +69,8 @@ export function toggleWindowSideBar(windowId) {
  * @param  {String} panelType
  * @memberof ActionCreators
  */
-export function toggleWindowSideBarPanel(windowId, panelType) {
-  return { type: ActionTypes.TOGGLE_WINDOW_SIDE_BAR_PANEL, windowId, panelType };
-}
-
-/**
- * popOutCompanionWindow - action creator
- *
- * @param  {String} windowId
- * @param  {String} panelType The type of panel content to be rendered
- *                            in the companion window (e.g. info, canvas_navigation)
- * @param  {String} position The position of the companion window to
- *                           set content for (e.g. right, bottom)
- * @memberof ActionCreators
- */
-export function popOutCompanionWindow(windowId, panelType, position) {
-  return (dispatch, getState) => {
-    const { companionWindowIds } = getState().windows[windowId];
-    companionWindowIds.map(id => dispatch(removeCompanionWindow(id)));
-
-    const action = dispatch(addCompanionWindow({ content: panelType, position }));
-
-    const companionWindowId = action.id;
-    dispatch(updateWindow(windowId, { companionWindowIds: [companionWindowId] }));
-
-    dispatch(toggleWindowSideBarPanel(windowId, 'closed'));
-  };
+export function setWindowSideBarPanel(windowId, panelType) {
```

Comment:
This action can probably get refactored away by just looking up the current left-position-companion-window from state instead.